web/src/App.less

```diff
@@ -125,6 +125,7 @@ img {
 .loginBackground {
   flex: auto;
   display: flex;
+  justify-content: center;
```

Comment:
Will this affect all entry pages, like "left", "right" won't work any more?

![image](https://user-images.githubusercontent.com/3787410/227671959-fa507afc-e85e-49e5-bf67-13bc49378296.png)